frontend/Interfaces/buddy/DIP/ImageContainer.h

```diff
@@ -244,8 +246,8 @@ Img<T, N> &Img<T, N>::operator=(const Img<T, N> &m) {
     this->allocated = ptr;
     this->aligned = ptr;
     this->_data = ptr;
-    return *this;
   }
```

Comment:
> 



> Because when reading the image is the default two-dimensional, there is no need to judge.



> Why delete if statement?

Because when reading the image is the default two-dimensional, there is no need to judge.

Reply:
> Why delete if statement?

Because when reading the image is the default two-dimensional, there is no need to judge.